cogitate_tool.py

```diff
@@ -0,0 +1,74 @@
+""" Command Line Interface for interacting with Github repository info. """
```

Comment:
I don't know if you guys are still gonna proceeding with the interactive CLI idea or using `Argparse`, but I guess it's fine to have these merged in if it's a working prototype. And changes and refactoring can be made later on.